<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>categoriesChangeDialog</class>
 <widget class="QDialog" name="categoriesChangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>94</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="categoriesNameLineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="categoriesNameLabel">
      <property name="text">
       <string>CategoryName</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="categoriesCodeLabel">
      <property name="text">
       <string>Code</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="categoriesAltNameLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="categoriesAltNameLabel">
      <property name="text">
       <string>AltName</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="categoriesCodeLineEdit">
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
      <property name="inputMask">
       <string>999999; </string>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
      <property name="cursorPosition">
       <number>6</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="categoriesButtonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>311</width>
     <height>26</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="categoriesIdLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>110</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="categoriesIdLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>110</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>categoriesButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>categoriesChangeDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>categoryNew()</slot>
  <slot>categoryUpdate()</slot>
 </slots>
</ui>
